invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js

```diff
@@ -121,6 +122,14 @@ class QueryBuilder {
     return this;
   }
 
+  withStartDate({ fromDate, toDate }) {
+    if (fromDate || toDate)
+      this.startDateQuery.push(
```

Comment:
I think in general the name should have the range as part of it so it can be more self explanatory...